ios/MdocDataTransfer.swift

```diff
@@ -0,0 +1,195 @@
+import MdocDataModel18013
+import MdocDataTransfer18013
+import MdocSecurity18013
+import React
+import SwiftCBOR
+
+@objc(MdocDataTransfer)
+class MdocDataTransfer: RCTEventEmitter {
+    let ON_RESPONSE_SENT_EVENT: String = "onResponseSent"
+    let ON_REQUEST_RECEIVED_EVENT: String = "onRequestReceived"
+
+    var bleServerTransfer: MdocGattServer?
+    var resolver: RCTPromiseResolveBlock?
+    var rejector: RCTPromiseRejectBlock?
+
+    override func supportedEvents() -> [String]! {
+        return ["onResponseSent", "onRequestReceived"]
+    }
+
+    // NFC is not enabled on iOS
+    @objc
+    func enableNfc() {
+        return
+    }
+
+    @objc
+    func initialize() -> String {
+        guard bleServerTransfer == nil else {
+            return ""
```

Comment:
This should throw an error

Reply:
DO you mean return an error string? I not mind it too much, but reinitialization calls can happen quite a bit with RN and that would make the UX quite annoying.